tfx_bsl/version.py

```diff
@@ -14,4 +14,4 @@
 """Contains the version string of tfx_bsl."""
 
 # Note that setup.py uses this version.
-__version__ = '0.28.0'
+__version__ = '0.28.1'
```

Comment:
don't remove the newline at the end of the file.

Reply:
Updated thanks PTAL